pkg/certificate/certificate.go

```diff
@@ -46,11 +46,12 @@ func NewKeyPair(tlsGenerate bool, keyFile, certFile string) *Keypair {
 	}
 }
 
-func (k *Keypair) generate() error {
-	org := ""
+func CreateRawKeyPair() (bytes.Buffer, bytes.Buffer, error) {
```

Comment:
Please add a comment as to what this function does and what the returned values are specifically.